lib/interpolation.ts

```diff
@@ -35,8 +35,8 @@ const delimiters: {
   },
 };
 
-type InterpolationData = {
-  [identifier: string]: string | number;
+export type InterpolationData = {
+  [identifier: string]: unknown;
```

Comment:
I opened this up so we don't need to do weird stuff like:
```typescript
i18n('some.key', {
  value: `${value}`
});  
```

These values are being cast to a string already, so these types should all be ok:

https://jsbin.com/bekepus/3/edit?js,console